frontend/src/scenes/persons/personsLogic.tsx

```diff
@@ -280,9 +280,16 @@ export const personsLogic = kea<personsLogicType<Filters, PersonLogicProps, Pers
                 } else if (activeTab && values.activeTab !== activeTab) {
                     actions.navigateToTab(activeTab as PersonsTabType)
                 }
+                if (personDistinctId) {
+                    // Decode the personDistinctId because it's coming from the URL, and it could be an email which gets encoded
+                    const decodedPersonDistinctId = personDistinctId && decodeURIComponent(personDistinctId)
 
-                if (personDistinctId && (!values.person || !values.person.distinct_ids.includes(personDistinctId))) {
-                    actions.loadPerson(personDistinctId) // underscore contains the wildcard
+                    if (
+                        personDistinctId &&
+                        (!values.person || !values.person.distinct_ids.includes(decodedPersonDistinctId))
+                    ) {
+                        actions.loadPerson(personDistinctId) // underscore contains the wildcard
+                    }
                 }
```

Comment:
- Nit: I'd put the `const decoded...` line to the top of the function, and even call the `_` argument `raw*` or `encodedPersonDistinctId`.
- Is it correct to call `loadPerson` with the encoded ID?

Reply:
> Nit: I'd put the const decoded... line to the top of the function, and even call the _ argument raw* or encodedPersonDistinctId.

Good call on this. Just re-read it, and 😬 it was messy - cleaned it up.

> Is it correct to call loadPerson with the encoded ID?

It works with either, but it's probably best to use the decoded one here, so updated to that.